File: code-ninja-course/JS/src/solid/Student.js
class Student {
	
	constructor(_emailAddress, _universityID) {
			this.universityID = _universityID;
			this.emailAddress = _emailAddress;
		}
	
	getEmailAddress() {
		throw "Not Implemented";
	}
	
	getUniversityID() {
		throw "Not Implemented";}
	
	getMonthlyEbookAllowance(){
		throw "Not Implemented";
	}
	
	getCurrentlyBorrowedEbooks() {
		throw "Not Implemented";
	}
	setMonthlyEbookAllowance(allowence) {
		throw "Not Implemented";
	}
	
	setCurrentlyBorrowedEbooks(books){
		throw "Not Implemented";
	}

}

File: code-ninja-course/C/src/solid/Student.c
#include <stdlib.h>
#include <string.h>
#include "Student.h"

struct Student* createStudent (const char* _emailAddress, int _universityID) {
	struct Student* student = malloc(sizeof(struct Student));
	student->universityID = _universityID;
	strcpy(student->emailAddress, _emailAddress);
	return student;
}

void setStudentMonthlybookAllowance(struct Student* student, int allowance) {

}


